src/casp/services/_settings.py

```diff
@@ -34,6 +34,7 @@ class AllEnvSettings(BaseSettings):
     LOG_LEVEL: str = "info"
     LOG_CONFIG: str = f"{SERVICES_DIR}/log_config.yaml"
     LOG_AS_JSON: bool = os.environ.get("LOG_AS_JSON", True)
+    MIN_CLIENT_VERSION: str = "1.4.0"
```

Comment:
We probably actually want this to be a constant since it's linked to the code (I don't think it would make sense to update this via config right?)

Reply:
That's a good point.  I'll make that change